docs/spec/WIP_provisioning/overview.md

```diff
@@ -0,0 +1,229 @@
+# Fee Distribution
+
+## Overview
+
+Fees are pooled separately and withdrawn lazily, at any time.  They are not
+bonded, and can be paid in multiple tokens.  An adjustment factor is maintained
+for each validator and delegator to determine the true proportion of fees in
+the pool they are entitled too.  Adjustment factors are updated every time a
+validator or delegator's voting power changes.  Validators and delegators must
+withdraw all fees they are entitled too before they can bond or unbond Atoms.
+
+## Affect on Staking
+
+Because fees are optimized to note
+
+Commission on Atom Provisions and having atoms autobonded are mutually
+exclusive (we can’t have both). The reason for this is that if there are atoms
+commissions and autobonding, the portion of atoms the fee distribution
+calculation would become very large as the atom portion for each delegator
+would change each block making a withdrawal of fees for a delegator require a
+calculation for every single block since the last withdrawal. Conclusion we can
+only have atom commission and unbonded atoms provisions, or bonded atom
+provisions and no atom commission
+
+## Fee Calculations
+
+Collected fees are pooled globally and divided out passively to validators and
+delegators. Each validator has the opportunity to charge commission to the
+delegators on the fees collected on behalf of the delegators by the validators.
+Fees are paid directly into a global fee pool. Due to the nature of of passive
+accounting whenever changes to parameters which affect the rate of fee
+distribution occurs, withdrawal of fees must also occur. 
+ 
+ - when withdrawing one must withdrawal the maximum amount they are entitled
+   too, leaving nothing in the pool, 
+ - when bonding, unbonding, or re-delegating tokens to an existing account a
+   full withdrawal of the fees must occur (as the rules for lazy accounting
+   change), 
+ - when a validator chooses to change the commission on fees, all accumulated 
+   commission fees must be simultaneously withdrawn.
+
+When the validator is the proposer of the round, that validator (and their
```

Comment:
Turn this sentence into an ordered list of steps separated by newlines: "1: That validator ..., 2: The reserve tax is then..."

Reply:
WIP, next PR